.github/workflows/tests.yml

```diff
@@ -0,0 +1,156 @@
+        sudo apt-get install -qqy selint
+        selint -V
+
+    - name: Create generated policy files
+      run: |
+        make conf
+        make generate
+
+    - name: Run file context checker
+      run: python3 -t -t -E -W error testing/check_fc_files.py
+
+    - name: Run SELint
+      run: |
+        # disable C-005 (Permissions in av rule or class declaration not ordered) for now: needs fixing
+        # disable W-005 (Interface call from module not in optional_policy block): refpolicy does not follow this rule
+        selint --source --recursive --summary --fail --disable C-005 --disable W-005 policy
+
+  build:
+    runs-on: ubuntu-latest
+
+    needs: lint
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        build-opts:
+          - {type: standard, distro: redhat, monolithic: y, systemd: y}
+          - {type: standard, distro: redhat, monolithic: n, systemd: y}
+          - {type: standard, distro: debian, monolithic: y, systemd: y}
+          - {type: standard, distro: debian, monolithic: n, systemd: y}
+          - {type: standard, distro: gentoo, monolithic: y, systemd: n}
+          - {type: standard, distro: gentoo, monolithic: n, systemd: n}
+          - {type: mcs, distro: redhat, monolithic: y, systemd: y}
+          - {type: mcs, distro: redhat, monolithic: n, systemd: y}
+          - {type: mcs, distro: debian, monolithic: y, systemd: y}
+          - {type: mcs, distro: debian, monolithic: n, systemd: y}
+          - {type: mcs, distro: gentoo, monolithic: y, systemd: n}
+          - {type: mcs, distro: gentoo, monolithic: n, systemd: n}
+          - {type: mls, distro: redhat, monolithic: y, systemd: y}
+          - {type: mls, distro: redhat, monolithic: n, systemd: y}
+          - {type: mls, distro: debian, monolithic: y, systemd: y}
+          - {type: mls, distro: debian, monolithic: n, systemd: y}
+          - {type: mls, distro: gentoo, monolithic: y, systemd: n}
+          - {type: mls, distro: gentoo, monolithic: n, systemd: n}
+          - {type: standard, distro: redhat, monolithic: y, systemd: y, apps-off: unconfined}
+          - {type: standard, distro: debian, monolithic: y, systemd: y, apps-off: unconfined}
+          - {type: standard, distro: gentoo, monolithic: y, systemd: n, apps-off: unconfined}
+          - {type: mcs, distro: redhat, monolithic: y, systemd: y, apps-off: unconfined}
+          - {type: mcs, distro: debian, monolithic: y, systemd: y, apps-off: unconfined}
+          - {type: mcs, distro: gentoo, monolithic: y, systemd: n, apps-off: unconfined}
+          - {type: mls, distro: redhat, monolithic: y, systemd: y, apps-off: unconfined}
+          - {type: mls, distro: debian, monolithic: y, systemd: y, apps-off: unconfined}
+          - {type: mls, distro: gentoo, monolithic: y, systemd: n, apps-off: unconfined}
+  
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.5
```

Comment:
Why is this specific version used? If there is a rationale of using the lowest supported version of Python, a comment could be added to help deciding in the future whether this version should be upgraded (and how)

Reply:
It's what we have in the travis test. IIRC it was the oldest Python they support. It's meant to be the minimum Python version needed to build (or at least as old as CI will support). I'll add a comment.